test/misc-components/syncTest.js

```diff
@@ -616,8 +616,7 @@ describe('Syncing bookmarks from an existing profile', function () {
     const folder = `[data-test-id="bookmarkToolbarButton"][title="${folderTitle}"]`
     yield Brave.app.client
       .waitForVisible(folder)
-      .click(folder)
-      .click(folder)
+      .doubleClick(folder)
```

Comment:
This change let the test passes locally, while the test fails on Travis:

https://travis-ci.org/brave/browser-laptop/jobs/277154849#L3775

````
  1) Syncing bookmarks from an existing profile update bookmark, moving it into the folder:
     Error: element (".contextMenu") still not visible after 10000ms
      at elements(".contextMenu") - isVisible.js:54:17
      at isVisible(".contextMenu") - waitForVisible.js:73:22
````

`npm run test -- --grep='Syncing bookmarks from an existing profile update bookmark, moving it into the folder'`

Reply:
@cezaraugusto @bsclifton would you mind helping me to fix the issue? thanks!

Reply:
for me they are all failing. I just did a rebase + push let's see how it goes